buildscripts/generate-release-contributors.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+last_version=$1
+auth_token=$2
+
+git log --pretty=format:"%s" $last_version..HEAD | grep -oE "\(#[0-9]{4,}\)" | grep -oE "[0-9]{4,}" | xargs -I{} curl -s -H "Authorization: token $auth_token" https://api.github.com/repos/open-telemetry/opentelemetry-java-instrumentation/issues/{} | grep login | sed 's/.*"login": "\([^"]*\)".*/\1/' | sort -uf > pr-authors
```

Comment:
@trask I'm good with going all in on expecting the `gh` CLI available. It also removes the need to deal with auth token in scripts.

It'll let you use `-q` for far more enjoyment than these sed expressions I think :)

e.g. https://cli.github.com/manual/gh_pr_view

Reply:
Also if interested in adventures in graphql-land, queries would look something like this one

```
gh api graphql --paginate -F q="repo:open-telemetry/opentelemetry-java-instrumentation is:pr is:merged merged:>=2021-11-26" -f query='

query($q: String!, $endCursor: String) {
  search(query: $q, type: ISSUE, first: 100, after: $endCursor) {
    edges {
      node {
        ... on PullRequest {
          author { login }
        }
      }
    }
  }
}
' -q '.data.search.edges.[].node.author.login' | sort -uf
```

Reply:
> I'm good with going all in on expecting the `gh` CLI available. It also removes the need to deal with auth token in scripts.

sounds good, I'll try this out and submit generate-release-contributors.sh in a separate PR